docs/start-new-project-1/create-required-services.md

```diff
@@ -40,8 +40,9 @@ module.exports = function (app) {
   app.configure(services.files); // mongoose service to manage files collection, uploads files with upload service
   app.configure(services.dashboard); // A dashboard for managing your app
   app.configure(services.userAbilities); // Provides user permissions information
-  // Specific project services
-  app.configure(posts);
+
+  // Specific project services (we will create the posts service later during this guide)
+  // app.configure(posts);
```

Comment:
..when you follow the docs you will get an "posts not defined" probleme at the step where the reader should start the dev server the first time